h/templates/client/viewer.html

```diff
@@ -1,15 +1,17 @@
-<selection-tabs ng-if="feature('selection_tabs') && !search.query() && selectedAnnotationCount() === 0"
+<selection-tabs ng-if="feature('selection_tabs') && !search.query() && selectedAnnotationCount() === 0 && !waitingToAnchorAnnotations"
```

Comment:
Hiding the tabs completely until anchoring completes means that there is a flicker where all the tabs disappear when switching groups.

I think that it will look better if the tab bar and Annotations / Page Notes tabs are always shown, but the Orphans tab is only shown if anchoring is complete AND there are orphans.